test/index.spec.js

```diff
@@ -0,0 +1,72 @@
+import keyboardOnlyOutlines from '../';
+
+describe('keyboard.only-outlines', () => {
+    let dispose;
+
+    afterEach(() => {
+        dispose();
+    });
+
+    it("should add a style node to the document's head", () => {
```

Comment:
This should be `'` instead of `"`. Isn't this throwing linting errors? hmmm

Reply:
In fact, this shouldn't been letting you commit. Is husky setup correctly or are you using --no-verify when committing?